pkg/volume/csi/csi_plugin_test.go

```diff
@@ -226,18 +226,17 @@ func TestPluginNewUnmounter(t *testing.T) {
 	pv := makeTestPV("test-pv", 10, testDriver, testVol)
 
 	unmounter, err := plug.NewUnmounter(pv.ObjectMeta.Name, testPodUID)
-	csiUnmounter := unmounter.(*csiMountMgr)
 
 	if err != nil {
 		t.Fatalf("Failed to make a new Unmounter: %v", err)
 	}
-
-	if csiUnmounter == nil {
+	if unmounter == nil {
 		t.Fatal("failed to create CSI Unmounter")
 	}
+	csiUnmounter := unmounter.(*csiMountMgr)
```

Comment:
Correct me if I'm wrong, but I'm not sure that moving this in this way actually changes anything. The logic is still exactly the same except now the error message on line 234 is not accurate.